lib/sycamore/sycamore/connectors/opensearch/opensearch_reader.py

```diff
@@ -141,6 +141,10 @@ def to_docs(self, query_params: "BaseDBReader.QueryParams") -> list[Document]:
 
             result = list(unique_docs.values())
 
+            # sort elements per doc
+            for doc in result:
+                doc.elements.sort(key=lambda e: e.element_index if e.element_index is not None else float("inf"))
```

Comment:
Does the shorter form not work?

`doc.elements.sort(key=lambda e: e.element_index or float("inf"))`

Reply:
element_index == 0 gets dropped that way

Reply:
or pushed to the end rather